service/service.go

```diff
@@ -22,6 +22,7 @@ type Service struct {
 	Teams        []string
 	Doc          string
 	IsRestricted bool `bson:"is_restricted"`
+	Instances    []ServiceInstance
```

Comment:
I'm -1 for to add this field.

I believe that the better approach is to add a `service.FindServiceInstanceByName("name", &user)` method and to split this handler in two handlers, one to get the service info and another to get service instances by service name.